client/state/invites/reducer.js

```diff
@@ -138,7 +148,10 @@ export const links = withSchemaValidation( inviteLinksSchema, ( state = {}, acti
  * @returns {Array}                  Updated array of invite objects.
  */
 function deleteInvites( siteInvites, invitesToDelete ) {
-	return siteInvites.filter( ( siteInvite ) => ! includes( invitesToDelete, siteInvite.key ) );
+	if ( ! Array.isArray( siteInvites ) || ! Array.isArray( invitesToDelete ) ) {
+		return siteInvites ?? [];
+	}
```

Comment:
We were calling `siteInvites.filter()`  before, so it must have been an array. 

The `Array.isArray( siteInvites ) ` check should not be necessary. 

Same for the nullish coalescing on the return value.

```suggestion
	if ( ! Array.isArray( invitesToDelete ) ) {
		return siteInvites;
	}
```